tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -14,10 +15,40 @@
 from marionette_driver.errors import (NoSuchElementException,
                                       StaleElementException,
                                       InvalidResponseException)
+import mozfile
 
 from environment import GaiaTestEnvironment
 from file_manager import GaiaDeviceFileManager, GaiaLocalFileManager
 
+DEFAULT_SETTINGS = {
+    'airplaneMode.enabled': False,  # disable airplane mode
+    'audio.volume.alarm': 0,  # mute alarm audio
+    'audio.volume.content': 0,  # mute content audio
+    'audio.volume.notification': 0,  # mute audio notifications
+    'camera.sound.enabled': False,  # mute camera sounds
+    'edgesgesture.enabled': False,  # disable edge gestures
+    'ftu.manifestURL': None,  # disable the first time usage app
+    'keyboard.autocorrect': False,  # disable auto-correction of keyboard
+    'keyboard.clicksound': False,  # mute keyboard click sound
+    'keyboard.enabled-layouts': str({
+        'app://keyboard.gaiamobile.org/manifest.webapp': {
+            'en': True, 'number': True}}),  # reset keyboard layouts
+    'keyboard.vibration': False,  # disable keyboard vibration
+    'language.current': 'en-US',  # reset language to en-US
+    'lockscreen.enabled': False,  # disable lockscreen
```

Comment:
I just edited the last comment to acknowledge that you did cover the GaiaDevice case. Mentioning so you don't just read the email. :)